### ISSUE TITLE
Error in Training loss 

### ISSUE THREAD

**Issue by ManishSoni1908** [dvornikita/blitznet#29]
Hey,
Thanks for releasing the awesome work on instance segmentation.
I am inspired from your work and just want to check out the results.

I started training on pascal Voc 2012 data-set with extra annotations. Algorithm combined loss starts from 14 and after 50k iterations it drop to 2-3 and after that it is just fluctuating between these values, even i tried to train it for more iterations but no drop in loss. 

Can you please suggest what happening.
I am using Geforce GTX 1080 with 8 gb GPU Ram and training the algorithm with batch_size 16.

If possible can you provide the Pre-trained model on any dataset either pascal or coco, so that i can check out the performance.

Thanks

**Comment by CarryJzzZ** [comment]
hi, @ManishSoni1908 
Firstly, this project does not work on instance segmentation, it combines object detection and semantic segmentation, the results can be improved on both tasks via joint training.
The training loss 2-3 does not indicates the bad performance,  you can use the model currently to test mIoU or mAP. and then you can judge the performance.
Finally,  [https://github.com/dvornikita/blitznet#testing-a-model](url) has provided the pre-trained model, you can use these to finetune or test.

**Comment by dvornikita** [comment]
@ManishSoni1908 , @CarryJzzZ is right. Just follow the testing-a-model section to download the weights and run a trained model.
@CarryJzzZ , thanks for the response.

**Comment by dvornikita** [comment]
Closing due to lack of recent activity.
